app/src/main/java/com/eveningoutpost/dexdrip/Tables/CalibrationDataTable.java

```diff
@@ -118,7 +118,7 @@ public boolean onLongClick(View v) {
                         public void onClick(DialogInterface dialog, int which) {
                             switch (which){
                                 case DialogInterface.BUTTON_POSITIVE:
-                                    calibration.invalidate();
```

Comment:
this maybe has to be approached with caution, but tracing it through it should be an improvement. Possibly as it is from interactive it should have the from_interactive set to true but we can leave that for now.